whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/attrs/AsBlockRange.java

```diff
@@ -60,4 +61,17 @@ public long getBegin() {
     public long getEnd() {
         return end;
     }
+
+    public String getEndWithPrefix() {
+        return String.format(AS_PREFIX_FORMAT, end);
+    }
+
+    public String getBeginWithPrefix() {
+        return String.format(AS_PREFIX_FORMAT, begin);
+    }
+
+    @Override
+    public String toString() {
+        return String.join("-", getBeginWithPrefix(), getEndWithPrefix());
```

Comment:
This is inconsistent with the original input (which has a space before and after the dash)